packages/nhsuk-frontend/src/nhsuk/components/card/_index.scss

```diff
@@ -252,7 +252,7 @@ $card-border-hover-colour: $nhsuk-card-border-hover-colour;
       color: $nhsuk-reverse-text-colour;
       background-color: nhsuk-colour("black");
 
-      a {
+      a:not(.nhsuk-action-link) {
```

Comment:
Ahh we do something similar for button groups via `a:not(.nhsuk-button)`

Could you please check if buttons-as-links aren't incorrectly given underlines too? 😬